### PR TITLE
 parameterize LMS endpoints

### PR DESCRIPTION
**Pull request by happythenewsad**
## WHAT
- parameterize LMS endpoints
- light refactoring
## WHY
so that each go environment connects to its respective LMS environment

### Screenshots
(If applicable. Also, please censor any sensitive data)

### Notion Card Links
(Please provide links to any relevant Notion card(s) relevant to this PR.)

PR Checklist | Your Answer
------------ | -------------
Have you added and/or updated tests? |  yes
Have you deployed to Staging? | yes
Self-Review: Have you done an initial self-review of the code below on Github? |
Design Review: If applicable, have you compared the coded design to the mockups? | n/a

### CODE REVIEW

File: services/comprehension/feedback-api-main/src/endpoint/endpoint.go [hunk 1]
```diff
@@ -54,8 +32,55 @@ var client = &http.Client {
...
+
+	var lms_domain = "https://www.quill.org"
+
+	if len(maybe_domain) > 0 {
+		lms_domain = maybe_domain
+	}
+	return lms_domain
+}
+
+func AssembleUrls() ([8]string) {
```
Comment (happythenewsad) [inline services/comprehension/feedback-api-main/src/endpoint/endpoint.go:51]: this is kinda gross

File: services/comprehension/feedback-api-main/src/endpoint/endpoint.go [hunk 2]
```diff
@@ -10,36 +10,14 @@ import (
...
-	plagiarism_api = "https://www.quill.org/api/v1/comprehension/feedback/plagiarism.json"
-	sentence_structure_regex_api = "https://www.quill.org/api/v1/comprehension/feedback/regex/rules-based-1.json"
-	post_topic_regex_api         = "https://www.quill.org/api/v1/comprehension/feedback/regex/rules-based-2.json"
-	typo_regex_api               = "https://www.quill.org/api/v1/comprehension/feedback/regex/rules-based-3.json"
-	spell_check_local            = "https://us-central1-comprehension-247816.cloudfunctions.net/spell-check-cloud-function"
-	spell_check_bing             = "https://www.quill.org/api/v1/comprehension/feedback/spelling.json"
-	batch_feedback_history_url   = "https://www.quill.org/api/v1/feedback_histories/batch.json"
-	automl_index                 = 3
-)
+var automl_index = 3
```
Comment (anathomical) [inline services/comprehension/feedback-api-main/src/endpoint/endpoint.go]: Can we not keep this as a `const`?

**Review by anathomical — CHANGES_REQUESTED** [review]
Looks like a pretty clean set of changes.  Might be worth adding some unit tests for the URL generating functions just to clearly communicate expectations?  Could probably be convinced that we shouldn't wait on those, but I imagine they're pretty simple tests.

File: services/comprehension/feedback-api-main/src/endpoint/endpoint.go (hunk 2 above)
Comment (happythenewsad) [reply services/comprehension/feedback-api-main/src/endpoint/endpoint.go]: yes

**Review by anathomical — APPROVED** [review]

**Pull request merged**